sr_unix/ydbinstall.sh

```diff
@@ -136,10 +136,10 @@ help_exit()
     echo "version is defaulted from mumps file if one exists in the same directory as the installer"
     echo "This version must run as root."
     echo ""
-    echo "Example usages are (assumes latest YottaDB release is r1.10 and latest GT.M version is V6.3-002)"
-    echo "  ydbinstall.sh                          # installs latest YottaDB release (r1.10) at /usr/local/lib/yottadb/r110"
-    echo "  ydbinstall.sh --utf8 default           # installs YottaDB release r1.10 with added support for UTF-8"
-    echo "  ydbinstall.sh --installdir /r110 r1.10 # installs YottaDB r1.10 at /r110"
+    echo "Example usages are (assumes latest YottaDB release is r1.20 and latest GT.M version is V6.3-002)"
+    echo "  ydbinstall.sh                          # installs latest YottaDB release (r1.20) at /usr/local/lib/yottadb/r120"
+    echo "  ydbinstall.sh --utf8 default           # installs YottaDB release r1.20 with added support for UTF-8"
+    echo "  ydbinstall.sh --installdir /r120 r1.20 # installs YottaDB r1.20 at /r120"
     echo "  ydbinstall.sh --gtm                    # installs latest GT.M version (V6.3-002) at /usr/local/lib/fis-gtm/V6.3-002_x86_64"
```

Comment:
Are the V6.3-002 comments correct? I think we are doing V6.3-003A now right? I haven't looked deeper but assume there are other V6.3-002 references in here that should probably be corrected.

Reply:
Fixed. There were just a couple of V63002 references that are now fixed to be V63003A.